pkg/storage/sqlstorage/migrates/17-optimized-segments/postgres.sql

```diff
@@ -0,0 +1,30 @@
+--statement
+drop trigger if exists log_entry on "VAR_LEDGER_NAME".log;
+drop trigger if exists volumes_changed on "VAR_LEDGER_NAME".log;
```

Comment:
The code use "--statement" to split statements.
Seems to works like this. But maybe we should keep the same behavior.

Reply:
you're right :)